doc/guide/cli_quickstart.hpp

```diff
@@ -95,7 +95,7 @@ other tools; for instance, we can easily calculate the number of points that
 were predicted incorrectly:
 
 @code{.sh}
-$ diff -U 0 predictions.csv covertype-test.labels.csv | grep '^@@' | wc -l
+$ diff -U 0 predictions.csv covertype.test.labels.csv | grep '^@@' | wc -l
```

Comment:
Actually, shouldn't this be `covertype-small.test.labels.csv` given the `mlpack_random_forest` call above?  I agree it should be changed though.

Reply:
Oops, you're right. I have added a new commit, you can either squash and merge or I can amend my commit depending on your preference. Thanks